lista znaków towarowych/LZT.R

```diff
@@ -0,0 +1,21 @@
+library(readr)
+library(assertr)
+library(dplyr)
+listaZnakowTowarowych <- read_delim("data/2018_10_26_listaZnakowTowarowych.csv", ";", escape_double = FALSE, trim_ws = TRUE)
+
+validator <- Validator$new()
+
+validation_rules <- function(data) {
```

Comment:
We don't want to use the funciton here as it complicates the pipeline that we want to show on presentation.
Please add content of the function to the below pipeline.

Reply:
done